### DISCUSSION TITLE
Response to bathymetry edits

### DISCUSSION THREAD

**Discussion by fobryan3** [NCAR/omwg_dev#55] [General]
I have run one cycle of a forced ocean-ice case with changes in the bathymetry a number of channels (Denmark St., Faroe Bank Channel, Florida Sts., and Caribbean passages, etc). Here is a quick first look at the response in the AMOC averaged over the final 30 years. This is from my own quick and dirty AMOC code. I need to get mom6-tools working for me again to make a complete comparison. 

From this very superficial look the changes in the circulation are modest but positive with a deepening of the NADW cell north of about 20N.
<img width="896" alt="Screenshot 2024-10-15 at 3 39 37 PM" src="https://github.com/user-attachments/assets/fbc62df5-bfce-4b74-8742-21fb078baa8b">
<img width="925" alt="Screenshot 2024-10-15 at 3 27 52 PM" src="https://github.com/user-attachments/assets/bb90e1db-15c9-4971-9b09-59671911376c">

The changes are also apparent but a little more difficult to discern in the sigma2 coordinate overturning.
<img width="884" alt="Screenshot 2024-10-15 at 2 49 56 PM" src="https://github.com/user-attachments/assets/fcf5e5a5-20a4-44bf-8886-74859308463f">
<img width="890" alt="Screenshot 2024-10-15 at 3 03 56 PM" src="https://github.com/user-attachments/assets/cb0a08ad-1ec3-47ca-b478-efc8caa14837">

**Comment by gustavo-marques** [comment]
Here are some AMOC plots for these simulations:

![image](https://github.com/user-attachments/assets/6a965a9e-0aee-4b0a-90cf-ced96770b48e)
![image](https://github.com/user-attachments/assets/39233a8d-798d-4a09-9075-163675b88069)

![image](https://github.com/user-attachments/assets/405d2d8c-b53c-45b8-9c0d-b0150c6aad74)
![image](https://github.com/user-attachments/assets/2c331b4f-0598-4f4d-aa0b-697465f75d62)
![image](https://github.com/user-attachments/assets/55e74a74-07d3-454d-941f-eb9d07360534)

These bathymetric changes have a significant effect on the AMOC. Interestingly, the profile at 26 N does not show a southward flow. Perhaps we need to run 79.topoed for longer to see this.

**Reply by fobryan3** [reply]
I have now run the case above (now referred to as 2024.079.topoed.v1) for two cycles. The differences shown above are robust. Based on the above and additional diagnostics from Gustavo, it appears that too much Faroe-Iceland overflow water was spilling out of the trough into the Porcupine Abyssal Plain rather than exiting through the Faroe Bank Channel near the mid-Atlantic Ridge.
Control:

<img width="1102" alt="Screenshot 2024-11-07 at 10 44 19 AM" src="https://github.com/user-attachments/assets/43030dc4-4a43-4c3a-b3f5-78981d7bb7e7">

topoed.v1:

<img width="1108" alt="Screenshot 2024-11-07 at 10 50 40 AM" src="https://github.com/user-attachments/assets/8a55324e-4272-4e89-9225-353c444ea356">


A second case (2024.079.topoed.v2) with less drastic deepening of the northern sills (Denmark and Faroe Bank) was run for one cycle. This case seems to be a compromise between the control and topoed.v1, recovering some northward bottom water flow at 26.5N. It still extends to a shallower depth than RAPID, but is an improvement over the control.
topoed.v2

<img width="914" alt="Screenshot 2024-11-12 at 3 01 52 PM" src="https://github.com/user-attachments/assets/284d4b76-937b-438d-95d8-339b0d9c55b8">


<img width="666" alt="Screenshot 2024-11-12 at 1 29 23 PM" src="https://github.com/user-attachments/assets/2d4ef38c-4e72-4e06-93f6-84f1caf3d01a">

The RAPID timeseries shows the two topoed cases close to each other through the 2000's with the drop in the 2010's  sensitive to the topography.

<img width="844" alt="Screenshot 2024-11-12 at 3 04 09 PM" src="https://github.com/user-attachments/assets/26c01cc6-efdd-4824-a661-de4732a86317">

I am now running two more cases. The first (2024.079.topoed.v3) deepens some topography at the southern entrance to the Ceara Absyssal Plain off of Brazil. There is a chain of shallow seamounts in this area that made the smoothed topography quite a bit shallower than the median or max depth for those cells, potentially blocking the northward flow of AABW. Initial results (~15 years) don't look like it had much impact.

The final case (20024.079.topoed.v4) is a further modification of Gibraltar. The first round of edits had a single grid point wide sill with the depth set to the observed value (284m). However, the faces of that cell (u-points) were quite a bit deeper, so the flow was coming across the sill from too deep (and salty) and too strong. I have the sill depth set over two points now, hoping it will cut down on our excessive Med outflow. Only have one year so far but the velocity response looks promising.

**Reply by fobryan3** [reply]
I ran a total of  five cases with the last two focused on trying to improve the flow of AABW through the South Atlantic and the Med Outflow. Here is a summary of the topography edits across cases. (colors change when something was changed in he topography for that feature, white means using the default topo).:
[Workhorse Topography Edits Fal 2024 - Google Sheets.pdf](https://github.com/user-attachments/files/17997442/Workhorse.Topography.Edits.Fal.2024.-.Google.Sheets.pdf)
The upshot is that I was only able to achieve minor improvements with both significantly biased. The Gibraltar outflow still results in a over expression of the Med salt tongue as seen in the T-S diagram:
<img width="465" alt="Screenshot 2024-12-03 at 11 33 50 AM" src="https://github.com/user-attachments/assets/08efc0f3-f694-431a-92db-141337d54d84">
The salt tongue is at about the right density(~27.75) but is too warm and too salty. Maybe too little entrainment with incoming Atlantic water at the sill?

The AABW circulation still looks too weak with warming of the bottommost layers of Brazil Basin and especially in the basins in the Western Atlantic to the north. 
<img width="930" alt="Screenshot 2024-12-03 at 11 43 44 AM" src="https://github.com/user-attachments/assets/8597e54a-49c8-47a0-8f1b-4cd675985568">

The last round of changes tried to open up the deep flow paths, but I hypothesize that our vertical resolution of the bottom layers is insufficient to maintain the cold fresh signal of AABW. However, there is a modest improvement in the AMOC at 26N relative to RAPID with a weak deep cell and somewhat deeper southward flow of lower NADW, while maintaining a healthy overall NADW cell strength.
<img width="576" alt="Screenshot 2024-12-03 at 11 47 25 AM" src="https://github.com/user-attachments/assets/ceb567f6-b159-4bd6-87c5-cee73c4254d9">
<img width="620" alt="Screenshot 2024-12-03 at 11 49 58 AM" src="https://github.com/user-attachments/assets/b28a1a51-abfa-4b68-bf63-e1d0e647a0f3">

**Comment by fobryan3** [comment]
Final (?) Comment

I took one last pass (topoed.v6) at tweaking the Gibraltar channel widths over three cell faces bounding the Strait. This slightly improved the hydrography in the Salt Tongue region.
<img width="619" alt="Screenshot 2024-12-16 at 2 14 01 PM" src="https://github.com/user-attachments/assets/a8da34e0-a4fc-4a49-b2d0-6c6534bc2500" />

In all cases, the salinity maximum is still too warm and salty and tends to be too shallow compared to obs. Cases that match the observed depth of the salinity maximum tend to have the worst salinity bias. Case v6 has the lowest salinity bias and a medium depth bias.
<img width="712" alt="Screenshot 2024-12-16 at 2 16 14 PM" src="https://github.com/user-attachments/assets/436f3802-50c9-46ce-bf0f-8603037acd50" />

The transport of the exchange flow through the Strait is at the lower end of observations around 0.6 Sv in case v6.
<img width="918" alt="Screenshot 2024-12-16 at 2 19 06 PM" src="https://github.com/user-attachments/assets/05f00dc8-15e1-4a5a-b10c-835d14f990f7" />

The problem is that the salinity of the outflow is too high and is not diluted enough by entrainment on the descent west of Gibraltar.
<img width="892" alt="Screenshot 2024-12-16 at 2 20 18 PM" src="https://github.com/user-attachments/assets/a00ba5eb-7060-4088-aa0a-bc992fe0e48a" />

BTW, I found that to correctly compute the transport weighted temperature and salinity I needed to save the section files in native coordinates. I verified that this does not break the mom6_tools transport scripts, so recommend that we make this the default.
